tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "target": "esnext",
     "module": "commonjs",
     "moduleResolution": "node",
-    "allowJs": true,
+    "allowJs": false,
```

Comment:
I assume that we want to remove this after the TS migration is done, right?

Reply:
Correct, it's a temporary solution to resolve issues with types being partially in /types and partially in /lib